src/utils.js

```diff
@@ -51,6 +52,27 @@ export function getMargin(
   }
 }
 
+const individualSides = ['top', 'right', 'bottom', 'left']
+const sideMaps = {
+  horizontal: ['right', 'left'],
+  vertical: ['top', 'bottom'],
+  all: individualSides,
+  none: [],
+}
+
+export function getSides(sides?: string = ''): Array<IndividualSides> {
+  const allSides = sides.split(' ').reduce((acc, current) => {
+    if (individualSides.includes(current)) {
+      return [...acc, current]
+    } else if (current in sideMaps) {
+      return [...acc, ...sideMaps[current]]
+    }
+    return acc
+  }, [])
+
+  return uniq(allSides)
+}
```

Comment:
This is the method you suggested @arahansen, now you can use things like: "top horizontal" "all" or "top left right". I think I'll change all margin/paddings to use this on a follow up PR to prevent this one from growing any further